src/main/resources/application.yml

```diff
@@ -65,12 +65,11 @@ nakadi:
   auth:
     plugin:
       factory: org.zalando.nakadi.plugin.auth.DefaultApplicationServiceFactory
-  flood.retry.after: 300 # seconds
----
-
-spring:
-  profiles: acceptanceTest
```

Comment:
Some tests required this profile. That might explain why some tests are broken now.